it/uk/gov/hmrc/gatekeeperemail/repositories/SentEmailRepositoryISpec.scala

```diff
@@ -95,11 +100,11 @@ class SentEmailRepositoryISpec extends AnyWordSpec with PlayMongoRepositorySuppo
   "findNextEmailToSend" should {
     "find the oldest pending email" in {
       val expectedNextSendRecipient = "old.email@digital.hmrc.gov.uk"
-      val emailsToSend              = List(sentEmail.head.copy(createdAt = LocalDateTime.now().minusMinutes(10), recipient = expectedNextSendRecipient))
+      val emailsToSend              = List(sentEmail.head.copy(createdAt = now.minusSeconds(10 * 60), recipient = expectedNextSendRecipient))
       await(serviceRepo.persist(emailsToSend))
       await(serviceRepo.persist(sentEmail))
 
-      val Some(nextEmail) = await(serviceRepo.findNextEmailToSend)
+      val nextEmail = await(serviceRepo.findNextEmailToSend).get
```

Comment:
You could use the OptionValues handling of ScalaSpec - then you get test failure rather than exceptions

Reply:
Not quite what I meant because this can still fail pattern matching